mkdocs.yml

```diff
@@ -0,0 +1,50 @@
+site_name : Img AI App
+
+# theme: readthedocs
+theme:
+  name : material
+
+nav:
+  - Home Page : 'index.md'
+  - Open Source :
+    - Programs we are part of :
+      - info : '#'
+      - about : '#'
+      - SLOP : '#'
+      - SWOC : '#'
+      - DWOC : '#'
+
+    - Contribution Guide : 'CONTRIBUTING.md'
+    - Code of Conduct : 'CODE_OF_CONDUCT.md'
+    - License : '#'
+
+
+  - Getting Started :
+    -  Using this repo as a template on GitHub : '#'
+    - Getting the source code locally : '#'
+    - Model Integration : '#'
+    - Updating Model Files : '#'
+    - Updating the Image Classifier Script : '#'
+    - Front End Changes : '#'
+    - Home Page [ Of the app] - : '#'
+    - Team Page : '#'
+    - About Page : '#'
```

Comment:
All of these are broken links and useless. Undestand the difference between TOC for pages and a sitemap for the entire website